CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangelog.com/en/1.0.0/) from version 0.9 onwards.
 
+## 0.14 (unreleased)
+
+### Added
+- :tada: **Enhancement** Added DateTimeIndicator a new Indicator for dates.
+- :tada: **Enhancement** Added DayOfWeekRule for specifying days of the week to trade.
+- :tada: **Enhancement** Added TimeRangeRule for trading within time ranges.
+
 ## 0.13 (unreleased)
```

Comment:
Please change 'unreleased' to 'released November 5, 2019'

Reply:
Done